docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+services:
+  postgres:
+    image: postgres
```

Comment:
Can we pin to a specific version here please, it's bad practice to let this default to `latest`.